test/js/web/fetch/fetch-leak-test-fixture-5.js

```diff
@@ -66,6 +66,19 @@ function getBody() {
     case "urlsearchparams":
       body = getURLSearchParams();
       break;
+    case "iterator":
+      body = async function* iter() {
+        yield (cachedBody ??= getString());
+      };
+      break;
+    case "stream":
+      body = new ReadableStream({
+        start(c) {
```

Comment:
This should use pull instead of start and have a slight delay in pull. We may detect if the ReadableStream has already ended with a single chunk and do something else